src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -256,16 +256,19 @@ const formConfig = {
       }
     },
     ratedDisabilities: {
-      title: 'Your Rated Disabilities',
+      title: (isReview) => {
+        if (isReview) {
+          return 'Rated Disabilities';
+        }
+        return 'Your Rated Disabilities';
```

Comment:
Is this backwards? (Not sure when we'd want to say "Your"...)
![image](https://user-images.githubusercontent.com/12970166/40318261-dfb3f002-5cd8-11e8-8765-4c32a956b313.png)

Reply:
@peggygannon recommend omitting "Your" on the review page.